Userland/Libraries/LibJS/Runtime/RegExpLegacyStaticProperties.h

```diff
@@ -14,7 +14,7 @@
 namespace JS {
 
 // https://github.com/tc39/proposal-regexp-legacy-features#regexp
-// The %RegExp% instrinsic object, which is the builtin RegExp constructor, has the following additional internal slots:
+// The %RegExp% intrinsic object, which is the builtin RegExp constructor, has the following additional internal slots:
```

Comment:
This is copied from the spec text in case you feel like making more PRs :makemore:

Reply:
sure, https://github.com/tc39/proposal-regexp-legacy-features/pull/20

Reply:
…but, https://github.com/tc39/proposal-regexp-legacy-features/pull/18, so who knows if that'll ever get merged